.gitignore

```diff
@@ -8,3 +8,21 @@ build
 data
 dist
 venv/
+
+.vscode/
+
+
+# Jetbrains Stuff
```

Comment:
Not sure if all these IDE and Windows lines should be included in the repo .gitignore... they should be ignore locally IMHO

Reply:
MacOS ones are ATM, so I was putting the windows set (from Github's recommendations) in just in case. I don't necessarily see any harm. I can try to think if there is a better way, probably within subfolders. I will also clean up the Jetbrains stuff.

Reply:
Point taken for the windows stuff! :+1:

Reply:
Please keep the Windows stuff (`thumbs.db`, `desktop.ini`) but clear the VSCode/JetBrains stuff since it's too specific.